MinecraftClient/Translations.cs

```diff
@@ -7,17 +7,20 @@
 namespace MinecraftClient
 {
     /// <summary>
-    /// Allows to localize the app in different languages
+    /// Allows to localize MinecraftClient in different languages
     /// </summary>
     /// <remarks>
     /// By ORelio (c) 2015-2018 - CDDL 1.0
     /// </remarks>
     public static class Translations
     {
         private static Dictionary<string, string> translations;
+        private static string translationFilePath = "lang" + Path.DirectorySeparatorChar + "mcc";
+        private static bool debugMessages = true; // Settings.LoadSettings have not been called yet at the time I guess. 
+                                                  // Hence Settings.DebugMessages will always return false
```

Comment:
Indeed. You may explicitly call `LoadTranslationsFile` from `Program.Main` instead of using the `static` constructor.

Reply:
Okay.